packages/mask/src/plugins/Savings/SNSAdaptor/SavingsForm.tsx

```diff
@@ -41,6 +47,11 @@ export interface SavingsFormProps {
     onClose?: () => void
 }
 
+const ProtocolName = {
+    [ProtocolType.Lido]: 'lido',
+    [ProtocolType.AAVE]: 'aave',
```

Comment:
```suggestion
    [ProtocolType.Lido]: 'Lido',
    [ProtocolType.AAVE]: 'AAVE',
```

Reply:
Recommend to use `createLookupTableResolver`, you will never lost new `ProtocolType`s.